core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -227,13 +226,28 @@ class Analyzer(
         scalaCompiler.askCompletionsAt(pos(fileInfo, point), maxResults, caseSens)
       } pipeTo sender
     case UsesOfSymbolAtPointReq(file, point) =>
-      import context.dispatcher
-      val response = if (toSourceFileInfo(file).exists()) {
+      if (toSourceFileInfo(file).exists()) {
+        val p = pos(file, point)
+        scalaCompiler.askLoadedTyped(p.source)
+        scalaCompiler.askSymbolFqn(p) match {
+          case Some(fqn) =>
+            indexer forward FindUsages(fqn.fqnString)
+          case None =>
+            sender ! FalseResponse
```

Comment:
this seems to be repeating a lot of code... can't we just treat this like a two part query as we do for the Doc requests? i.e. higher up sends a TypeAtPoint and then reroutes it to the indexer. Means the analyzer doesn't need to know that these requests exist.